channeld/channeld.c

```diff
@@ -2381,38 +2393,57 @@ static void peer_reconnect(struct peer *peer,
 	}
 }
 
-/* Funding has locked in, and reached depth. */
-static void handle_funding_locked(struct peer *peer, const u8 *msg)
+/* ignores the funding_depth unless depth >= minimum_depth
+ * (except to update billboard, and set peer->confirmation_needed). */
+static void handle_funding_depth(struct peer *peer, const u8 *msg)
 {
 	unsigned int depth;
+	struct short_channel_id *scid;
 
-	if (!fromwire_channel_funding_locked(msg,
-					     &peer->short_channel_ids[LOCAL],
-					     &depth))
-		master_badmsg(WIRE_CHANNEL_FUNDING_LOCKED, msg);
+	if (!fromwire_channel_funding_depth(tmpctx,
+					    msg,
+					    &scid,
+					    &depth))
+		master_badmsg(WIRE_CHANNEL_FUNDING_DEPTH, msg);
 
 	/* Too late, we're shutting down! */
 	if (peer->shutdown_sent[LOCAL])
 		return;
+	if (depth < peer->channel->minimum_depth) {
```

Comment:
another nitpick, a space above this lines makes it easier to read :)

Reply:
Maybe make the `unsigned int depth` also an `u32` ?

Reply:
Thank you, your advice is great:)